libphal/phal_dump.C

```diff
@@ -87,9 +92,9 @@ struct DumpPIBMSRegVal {
 
 // PPE register details
 struct DumpPPERegValue {
-	uint16_t number;     // Register number
```

Comment:
do not make these formating changes as it is confusing

Reply:
resolved. avoiding clang.